search-service/src/main/kotlin/com/egm/stellio/search/config/SearchProperties.kt

```diff
@@ -4,5 +4,6 @@ import org.springframework.boot.context.properties.ConfigurationProperties
 
 @ConfigurationProperties("search")
 data class SearchProperties(
-    val payloadMaxBodySize: Int
+    val payloadMaxBodySize: Int,
+    var deleteEntityByCreator: Boolean
```

Comment:
Could be false by default. So you add the property is used only when you need the funcionnality

Reply:
in `search-service/src/main/resources/application.properties` it is `false` by default